app/timetables/tests/test_models.py

```diff
@@ -65,3 +65,63 @@ def test_meal_end_time_same_with_start_time_cannot_be_saved(self):
         )
 
         self.assertRaises(ValidationError, meal.save)
+
+
+class TimetableTest(TestCase):
+    """Tests the Timetable model."""
+
+    def setUp(self):
+        Timetable.objects.create(
+            name='timtable-item',
+            code='FBI23212',
+            api_key='419223',
+            cycle_length=1,
+            current_cycle_day=2,
```

Comment:
@andela-joyebanji if we have a cycle_length of 1 here, then having a current_cycle_day of 2 will not be feasible. Please refer to the wiki https://github.com/teamtaverna/core/wiki/Foodbot-Platform-Upgrade:-Taverna for better understanding of those fields.